js/api/Surveys.js

```diff
@@ -18,6 +18,26 @@ export function loadCachedSurveyList() {
   return realm.objects('Survey')
 }
 
+// Finds and returns the list of all surveys cached in Realm.io
+export function loadAllAcceptedSurveys(callback) {
+  loadAcceptedInvitations((err, invitations) => {
+    if (err) {
+      callback(err, [])
+      return
+    }
+
+    let surveys = []
+    const invitationsLength = invitations.length;
+    for (var i = 0; i < invitationsLength; i++) {
```

Comment:
Cleaning that up with the const.

On loops: I honestly prefer to use vars in for loops in ES6 due to their function scope, it's a habit that makes it easier to keep track of declarations and helps minimize bugs during development.

An edge-case example, which is silly for being allowed in ES6:
```
for (let i = 0; i < things.length; i++) {
  for (let i = 0; i < something_else.length; i++) {
    things[i].foo();
  }
}
```